### DISCUSSION TITLE
design of sensor #6

### DISCUSSION THREAD

**Discussion by Pitichinaccio** [srozum/film_camera_tester#26] [General]
I am just wondering about the design of sensor #6. 
In the wiki it says that it's "very similar by it's construction to sensor #2". 
The STL of the face plate has three immersions on the backside and holes to the front. So, this looks to me as there are three sensors in a diagonal row (like sensor #1). The PCB has only one TSL237 on it.
Just want to be sure that  there's only one sensor placed in the middle.
Maybe the STL for sensor #6 could just also be used as alternative housing for sensor #1?

Thanks, Bernd

**Comment by srozum** [comment]
Sensor 6 enclosure was designed as possible universal enclosure for all types of sensors. However PCBs for Sensor 1 or 2 which fit into this enclosure weren't designed.

This enclosure has few improvements compared to other versions:
- front plate can be clicked into place without glueing
- lid has special "ear" to fix the cable with zip tie
- medium format adapter easier to attach and remove

**Comment by ic-racer** [comment]
Thank you, I was also wondering about that.

**Comment by Pitichinaccio** [comment]
Thanks Serhiy!
